tests/integration/test_cdn_migration.py

```diff
@@ -1220,29 +1181,24 @@ def name_matcher(request):
     assert fake_requests.request_history[7].method == "GET"
     assert (
         fake_requests.request_history[8].url
-        == "http://localhost/v2/service_plan_visibilities?q=organization_guid%3Amy-org-id&q=service_plan_guid%3AFAKE-MIGRATION-PLAN-GUID"
+        == "http://localhost/v3/service_plans/FAKE-MIGRATION-PLAN-GUID/visibility/my-org-id"
     )
-    assert fake_requests.request_history[8].method == "GET"
+    assert fake_requests.request_history[8].method == "DELETE"
+
     assert (
         fake_requests.request_history[9].url
-        == "http://localhost/v2/service_plan_visibilities/my-service-plan-visibility"
+        == "http://localhost/v2/service_instances/asdf-asdf?purge=true"
     )
     assert fake_requests.request_history[9].method == "DELETE"
 
+    assert fake_requests.request_history[10].method == "PUT"
     assert (
         fake_requests.request_history[10].url
-        == "http://localhost/v2/service_instances/asdf-asdf?purge=true"
-    )
-    assert fake_requests.request_history[10].method == "DELETE"
-
-    assert fake_requests.request_history[11].method == "PUT"
```

Comment:
Are these deletions intentional?

Reply:
in API v2, service_plan_visibility was its own object with an ID - so the relationship between service_plan x and organization y had a service_plan_visibility z. In order to make x stop being visible in y you had to look up z then call DELETE on it, like DELETE /v2/service_plan_visibilities/z

Now, service_plans have a visibilities property that is a list of objects (representing orgs and spaces). In order to make x no longer visible on y you DELETE the org on the service plan like DELETE /v3/service_plans/x/visibility/y

These were the mocks for finding the value of `z`, so they're no longer needed